modules/mcc/src/wiener_filter.hpp

```diff
@@ -0,0 +1,45 @@
+/**
```

Comment:
where is the file taken from? What's the license?

Reply:
Done